models/form.js

```diff
@@ -4,14 +4,14 @@ var Schema = mongoose.Schema;
 var formSchema = Schema({
   _studentID: { type: Schema.Types.ObjectId, ref: 'Student' },
   period: String,
-  date: Date,
+  date: Number,
```

Comment:
Because that's what the viz needs to plot with - it is still generated as a dat object in index.js, but it is just saved as a number to make the plotting easier.